kernelci/db/kernelci_api.py

```diff
@@ -45,9 +45,10 @@ def _get(self, path):
         resp.raise_for_status()
         return resp
 
-    def _post(self, path, data=None):
+    def _post(self, path, params=None, data=None):
```

Comment:
The method is used in one place with data as a positional argument:
```python
                    resp = self._post(path, json.dumps(item))
```

So please keep the order of the arguments and add `params` at the end:
```suggestion
    def _post(self, path, data=None, params=None):
```

Reply:
Done.